app/controllers/events/view/speakers/list.js

```diff
@@ -90,9 +90,9 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
   }
 
   @action
-  viewSpeaker(speaker_id, event) {
-    const event_id = event.router.location.location.pathname.split('/')[2];
-    this.transitionToRoute('public.speaker.view', event_id, speaker_id);
+  viewSpeaker(speaker_id) {
```

Comment:
Why not pass the speaker directly?

Reply:
I think the way I have done now in latest commit looks fine. 
Please look into it.